.github/workflows/release.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Build Epinio
```

Comment:
The name should be different since it is `make release`

Reply:
It still mainly builds the binaries, we just combined all the cleanup commands under release. @jimmykarily what do you think?
I do not mind calling it release but it is not completely true.